### PR TITLE
alert counter to be initialized even the alert count is zero

### PR DESCRIPTION
**Pull request by GowthamShanmugam**
tendrl-bugid: Tendrl/node-agent#643

Signed-off-by: GowthamShanmugam <gshanmug@redhat.com>

### CODE REVIEW

**Comment by GowthamShanmugam** [comment]
@r0h4n @shtripat please review

**Comment by GowthamShanmugam** [comment]
Verified

File: tendrl/node_agent/manager/__init__.py [hunk 1]
```diff
@@ -90,7 +95,13 @@ def main():
     if NS.config.data.get("with_internal_profiling", False):
         from tendrl.commons import profiler
         profiler.start()
-
+    # Initialize alert count
+    try:
```
Comment (r0h4n) [inline tendrl/node_agent/manager/__init__.py]: This code needs  to be in node_agent/node_sync/__init__.py 

Not here

Comment (GowthamShanmugam) [reply tendrl/node_agent/manager/__init__.py]: ok

**Comment by GowthamShanmugam** [comment]
@r0h4n @shtripat changes done

**Comment by GowthamShanmugam** [comment]
@r0h4n @shtripat please review, changes done

**Pull request merged**